src/test/kotlin/com/workos/test/user_management/UserManagementApiTest.kt

```diff
@@ -1223,8 +1324,9 @@ class UserManagementApiTest : TestBase() {
         "revoked_at": null,
         "expires_at": "2021-07-01T19:07:33.155Z",
         "token": "Z1uX3RbwcIl5fIGJJJCXXisdI",
-        "accept_invitation_url": "https://myauthkit.com/invite?invitation_token=Z1uX3RbwcIl5fIGJJJCXXisdI",
+        "accept_invitation_url": "https://your-app.com/invite?invitation_token=Z1uX3RbwcIl5fIGJJJCXXisdI",
```

Comment:
makes sense to update this, more clear what the invitation url is redirecting to